tests/unit-test-run.mjs

```diff
@@ -0,0 +1,64 @@
+
+import serve from "./unit-test-server.mjs";
+import {
+  Builder,
```

Comment:
Please use 4-space indentation per coding style guideline.

Reply:
OK. Looking forward to auto formatting